<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mensageiro2_Cliente</class>
 <widget class="QMainWindow" name="Mensageiro2_Cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mensageiro - Trocador de mensagens via rede</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>USU√ÅRIO:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msgLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MENSAGEM:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="msgEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enviarButton">
    <property name="geometry">
     <rect>
      <x>734</x>
      <y>530</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ENVIAR</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="msgTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>525</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServidor">
    <property name="title">
     <string>Servidor</string>
    </property>
    <addaction name="actionConectar"/>
    <addaction name="actionDesconectar"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuMensagens">
    <property name="title">
     <string>Mensagens</string>
    </property>
    <addaction name="actionApagar"/>
   </widget>
   <addaction name="menuServidor"/>
   <addaction name="menuMensagens"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar...</string>
   </property>
  </action>
  <action name="actionDesconectar">
   <property name="text">
    <string>Desconectar</string>
   </property>
  </action>
  <action name="actionApagar">
   <property name="text">
    <string>Apagar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
